### PR TITLE
TJ-9611 Added more translation

### PR DESCRIPTION
**Pull request by KathiriyaAkshay**
TJ-9611 Added more translation

### CODE REVIEW

**Comment by PriteshKiri** [comment]
@KathiriyaAkshay can you please resolve the merge conflicts?

**Comment by KathiriyaAkshay** [comment]
Hey @PriteshKiri , resolved the merge conflicts.

**Comment by PriteshKiri** [comment]
Thanks @KathiriyaAkshay

File: frontend/src/_components/AppModal.jsx [hunk 1]
```diff
@@ -163,7 +161,6 @@ export function AppModal({
               style={{
                 borderColor: errorText ? '#DB4324 !important' : 'initial',
               }}
```
Comment (johnsoncherian) [inline frontend/src/_components/AppModal.jsx:163]: Any specific reason for removing `disabled` attribute from this button? If not we need to add it back

**Review by johnsoncherian — COMMENTED** [review]
@KathiriyaAkshay thanks for the contribution. Can you check the comments added and revert?

**Comment by PriteshKiri** [comment]
@KathiriyaAkshay can you please check the comments on the PR and also resolve the merge conflicts?

**Comment by PriteshKiri** [comment]
@KathiriyaAkshay are you still working on the issue?

**Comment by PriteshKiri** [comment]
Closing the PR!

**Pull request closed without merging**